test/python/unittest/API/APITest.py

```diff
@@ -1824,7 +1825,7 @@ def testIterator(self):
                 self.makeIteratorRoundTrip(b, backend_filesupport[b])
 
     def makeAvailableChunksRoundTrip(self, ext):
-        if ext == "h5":
+        if ext == "h5" or ext == "toml":
```

Comment:
What does not round-trip for toml (and h5)?

Reply:
`available_chunks()` is a backend-specific feature. In HDF5, it returns the entire dataset as there is no *native* way to retrieve defined regions from an HDF5 dataset (or if there is: I don't know about it and we are not using it).
In the JSON backend, we use a [rather crude algorithm](https://github.com/openPMD/openPMD-api/blob/99daca78e9ef2207b78e922ad393cadf9a8ab019/src/IO/JSON/JSONIOHandlerImpl.cpp#L303) to recover written chunks, based on where the dataset is `null` and where it is defined. (This is what I was recently referring to when I said that @eschnett's region calculus might be relevant also in our internal code).

In TOML however, there is no null, so this does not work.

Reply:
Ah right, this is chunk round-tripping. Sounds good!